cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -227,27 +227,17 @@ spec:
         #   value: "role=storage-node; storage=rook, ceph"
         # (Optional) CEPH CSI provisioner tolerations list. Put here list of taints you want to tolerate in YAML format.
         # CSI provisioner would be best to start on the same nodes as other ceph daemons.
-        # - name: CSI_PROVISIONER_TOLERATIONS
-        #   value: |
-        #     - effect: NoSchedule
-        #       key: node-role.kubernetes.io/controlplane
-        #       operator: Exists
-        #     - effect: NoExecute
-        #       key: node-role.kubernetes.io/etcd
-        #       operator: Exists
+        - name: CSI_PROVISIONER_TOLERATIONS
+          value: |
+            - operator: Exists
         # (Optional) Ceph CSI plugin NodeAffinity.
         # - name: CSI_PLUGIN_NODE_AFFINITY
         #   value: "role=storage-node; storage=rook, ceph"
         # (Optional) CEPH CSI plugin tolerations list. Put here list of taints you want to tolerate in YAML format.
         # CSI plugins need to be started on all the nodes where the clients need to mount the storage.
-        # - name: CSI_PLUGIN_TOLERATIONS
-        #   value: |
-        #     - effect: NoSchedule
-        #       key: node-role.kubernetes.io/controlplane
-        #       operator: Exists
-        #     - effect: NoExecute
-        #       key: node-role.kubernetes.io/etcd
-        #       operator: Exists
+        - name: CSI_PLUGIN_TOLERATIONS
```

Comment:
This will tolerate all taints including to run on masters?

Reply:
AFAIK, yes.